Source/FakeItEasy/Configuration/RecordingRuleBuilder.cs

```diff
@@ -14,8 +14,6 @@ public RecordingRuleBuilder(RecordedCallRule rule, RuleBuilder wrappedBuilder)
         {
             this.rule = rule;
             this.wrappedBuilder = wrappedBuilder;
-
-            rule.Applicator = x => { };

```

Comment:
But it worried me more, since the previous line was in test code, and this one is in production. Nevertheless, I can't see any harm in removing this. I tried to break the `NextCall.To` mechanism, but couldn't.